### ISSUE TITLE
Health detrimental figures are counted at twice the rate they should be

### ISSUE THREAD

**Issue by Fox-McCloud** [ParadiseSS13/Paradise#2575]
Basically, for each point of damage you take, it's like taking 2 damage--this means the crit icon shows up at 50 damage instead of 100--likewise, I do believe health detrimental effects are also taking this into consideration (slowdown, etc) as well.

@tigercat2000

**Comment by Spacemanspark** [comment]
This might explain why we get ahelps going "HELP I'M IN CRIT AHHHHHHH", we look at their stats and they've only taken 60 damage.

**Comment by Fox-McCloud** [comment]
Yeah; I noticed this after taking some minor tox damage (50), but was still moving super super slowly and had the crit icon, but wasn't in *actual* soft crit.

**Comment by tigercat2000** [comment]
I have no fucking idea how this is possible. Movement calculations are based on the health var, the health var is updated multiple times per Life()

**Comment by Fox-McCloud** [comment]
The bigger problem right now is the health bar being incorrect; while the movement is problematic, players thinking they're on death's doorstep when they only have a mere 50 damage is worse.

**Comment by Fox-McCloud** [comment]
Updating the post: it seems movement is actually fine; it's just the health HUD icon taht's messed up.

Health doll *appears* to be fine.
